app/models/student.rb

```diff
@@ -284,7 +285,7 @@ def create_user_account
     user = User.new do |u|
       u.first_name = first_name
       u.last_name = last_name
-      u.username = admission_no
+      u.username = first_name
```

Comment:
`first_name` as a user name might cause problem if we have `User.username` as a `unique` column. you might want to use `email_id` or `admission_no` or some unique combination.